bears/r/FormatRBear.py

```diff
@@ -25,7 +25,7 @@ def create_arguments(filename, file, config_file,
                          r_braces_on_next_line: bool=False,
```

Comment:
As you need to change this commit, you could fix some typos and stuff in the commit message:
> The `r_max_expression_length` option **passed** to
tidy_source is quite advanced ~~and difficult to
deal with, so it can be a good idea to disable it~~.
It is **disabled** when its value is `0` **[MISSING SPACE]**(the default
value) and ~~considered~~ **enabled** otherwise. **This commit disables this option by default.**